armi/bookkeeping/db/tests/test_comparedb3.py

```diff
@@ -184,7 +184,7 @@ def test_compareDatabaseSim(self):
                 dbs[1]._fullPath,
                 timestepCompare=[(0, 0), (0, 1)],
             )
-        self.assertEqual(len(diffs.diffs), 504)
+        self.assertEqual(len(diffs.diffs), 528)
```

Comment:
Do we love this test?  Is it useful to know when we make a PR that changes the number of diffs?

Reply:
Does it spark joy?

Reply:
I agree that it's pretty silly to just keep updating this number arbitrarily as things are changed in the database.

I think we want to verify that there are some contents in `diffs.diffs`, but maybe we just assert something like `len(diffs.diffs) > 1` and then it wouldn't need to be updated all the time. It's not perfect, but just a thought... open to suggestions.

Reply:
Okay, in the `main` branch I have updated this test to not be so fragile.